### ISSUE TITLE
python darknet.py classify gives different results vs command line

### ISSUE THREAD

**Issue by saraswat40** [AlexeyAB/darknet#6930]
I am trying to use the darknet.py/classify function for image classification. Unfortunately it gives different results from the command line option ( # darknet classifier predict ) for the same image.

Is there a solution to this? I am calling the classify like so:

```
inp_img = dn.load_image(fname.encode('ascii'), 0, 0)
res = dn.classify(self.net, self.meta, inp_img)
```

**Comment by saraswat40** [comment]
@Faultiness you will need to resize your images before giving it to darknet. On command line the resizing is done internally by darknet. In darknet.py you need to pass a resized image.

**Comment by Faultiness** [comment]
@saraswat40 Thanks for your reply!
I tried to resize the input image to the size of the network that I used to train for the model (which is 640*640) but the result doesn't change.

I reviewed the code of darknet_image.py and noticed the [Line 109](https://github.com/AlexeyAB/darknet/blob/master/darknet_images.py#L109) actually automatically resized the image before processing.

And, which might be really confused, I trained another model the Yolov4-custom then predict the same image with same python code (not modified) and the result is same with the command line output.
